.vib/argo-workflow-exec/goss/argo-workflow-exec.yaml

```diff
@@ -0,0 +1,8 @@
+command:
+  check-argo-workflow-exec:
+    exec: /opt/bitnami/argo-workflow-exec/bin/argoexec --help
+    exit-status: 0
```

Comment:
Why are we using `--help` instead of checking the version? I think that would be more valuable as a test.